### PR TITLE
Add stamped messages for GeoPoint and GeoPose.

### PR DESCRIPTION
**Pull request by dornhege**

### CODE REVIEW

File: geographic_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -17,6 +17,8 @@ add_message_files(
   GeographicMap.msg
   GeoPoint.msg
   GeoPose.msg
+  GeoPointStamped.msg
+  GeoPoseStamped.msg
```
Comment (mikepurvis) [inline geographic_msgs/CMakeLists.txt]: Alpha order please.

Comment (dornhege) [inline geographic_msgs/CMakeLists.txt]: Done.

File: geographic_msgs/msg/GeoPoseStamped.msg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Header header
+geographic_msgs/GeoPose geo_pose
```
Comment (mikepurvis) [inline geographic_msgs/msg/GeoPoseStamped.msg]: I'm on the fence about this, but you could consider following the `position` naming already established in a few other messages:

https://github.com/ros-geographic-info/geographic_info/search?q=GeoPoint

OTOH, the convention from geometry_msgs is more toward just naming the field after the type, which would suggest either `geo_pose` or just `pose`.

Comment (dornhege) [inline geographic_msgs/msg/GeoPoseStamped.msg]: I changed it to position/pose to be consistent within this package.

**Comment by mikepurvis** [comment]
LGTM.

I'll give a day in case @jack-oquin or others have concerns.

**Comment by jack-oquin** [comment]
LGTM.

What are your feelings about release distros? Under traditional maintenance guidelines, we would not add new messages to an old distro like Indigo, but could probably still add them to Kinetic. 

Given the long-term support posture of Indigo and the low risk involved in this update, I'd consider making an exception and releasing it there, too.

**Comment by mikepurvis** [comment]
For just adding stuff, it's a zero-risk thing, so I see no reason not to push a change like this out across the board.

**Comment by dornhege** [comment]
+1, we actually use this in indigo.

**Comment by jack-oquin** [comment]
No problem. I agree. I just didn't want to make that decision without any discussion.

If we release it to Indigo and Kinetic, I'll try to support Jade as well, assuming it's still possible to make a Jade release. I think it is.

**Comment by jack-oquin** [comment]
This seems like something we would have had in the beginning, if anybody had asked for it.

**Pull request merged**